dvc/command/experiments.py

```diff
@@ -1249,6 +1250,12 @@ def add_parser(subparsers, parent_parser):
         action="store_true",
         help="Remove all committed experiments.",
     )
+    remove_group.add_argument(
+        "-r",
+        "--git-remote",
+        metavar="<git_remote>",
+        help="Name of the Git remote to GC all of the experiment branches.",
```

Comment:
Why GC? Isn't this `exp remove`?

Reply:
Also what's the default? `origin`, I assume. Thanks

Reply:
There is no default, if `--git-remote` is not specified it removes a local experiment ref and not a remote one

Reply:
I think it is better to use "Name or URL of the Git remote to delete all of the experiment references."" 

> Also what's the default? origin, I assume. Thanks
No default for this

Reply:
> There is no default

What does `exp remove -g 1234567` do? Errors out? If so OK 👍 thanks